lib/fpclassify.c

```diff
@@ -0,0 +1,48 @@
+/* Replacement function for isnan/isinf/isfinite for processors using
+ * IEEE 754 floating point format.
+ * Copyright (C) 2017, bel2125
+ * License: LGPL v2 or MIT (as you like).
```

Comment:
Is your intention to provide this file with a different license than that of Check? Check uses LGPLv2.1. I do not think that there is a functional difference between v2 and v2.1, except that the license's name changed in v2.1.

Reply:
Do you mind if this file has the license which is mentioned in the other files, namely:
```
/*
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the
 * Free Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston,
 * MA 02110-1301, USA.
 */
```